docs/io/ascii/read.rst

```diff
@@ -859,3 +861,155 @@ them at the end.
 
   ..
     EXAMPLE END
+
+.. _io_ascii_how_to_examples:
+
+How to Find and Fix Problems Reading a Table
+============================================
+
+The purpose of this section is to provide a few examples how we can
+deal with tables that fail to read.
+
+Obtain the Data Table in a Different Format
+-------------------------------------------
+Sometimes it is easy to obtain the data in a more structured format that
+more clearly defines columns and metadata, e.g. a FITS or VO/XML table, or
+an ASCII table that uses a different colum separator (e.g. comma instead of
+white space) or fixed-width columns.
+In that case, the fastest solution can be to simply download or export the
+data again in a different format.
+
+Find the Problem
+----------------
+Usually, `astropy.io.ascii.read` tries many different formats until one
+succeeds in reading. If it works, that saves you from finding and
+setting right options for reading. However, if it fails to find any combination
+of format and format options that correctly parses the file, then you will get
+a long exception message which shows every format that was tried and ends
+with this advice::
+
+  ************************************************************************
+  ** ERROR: Unable to guess table format with the guesses listed above. **
+  **                                                                    **
+  ** To figure out why the table did not read, use guess=False and      **
+  ** fast_reader=False, along with any appropriate arguments to read(). **
+  ** In particular specify the format and any known attributes like the **
+  ** delimiter.                                                         **
+  ************************************************************************
+  
+To expand on this a bit, you probably know from looking at the file
+what format it is in, which must be one of the :ref:`supported_formats`.
+For instance maybe it is a basic space-delimited file but has the header
+line as a comment like below, which corresponds to the ``commented_header``
+format::
+
+  >>> table = """# name id
+  ...: Jill 1232
+  ...: Jack Johnson 456"""
+
+In order to find the actual problem with the reading this file, you would do::
+
+  >>> ascii.read(txt, format='commented_header', delimiter=' ', guess=False, fast_reader=False)
```

Comment:
You might need ellipses (sp?) directive for the doctest, if this is doctested, but let's wait for the CI to complain first after you fix the trailing whitespace.

p.s. You can set your IDE to auto-remove trailing whitespace, I think.